src/app/client/projects/desktop/src/app/modules/offline/components/offline-report-issues/offline-report-issues.component.ts

```diff
@@ -8,11 +13,46 @@ import { Component, OnInit } from '@angular/core';
 export class OfflineReportIssuesComponent implements OnInit {
   issueReportText = false;
   showNormalModal = false;
-  constructor() { }
-
+  descriptionCount: any;
```

Comment:
Cross-check with the naming convention of the method and properties of the class